contrib/storage-mongo/src/test/java/org/apache/drill/exec/store/mongo/MongoTestConstants.java

```diff
@@ -20,6 +20,7 @@
 public interface MongoTestConstants {
 
   public static final String LOCALHOST = "localhost";
+  // TODO: add some randomization to this as it fails when running concurrent builds
```

Comment:
will do. This was mainly a reminder for myself to open a jira before submitting.